Modules/ReportGenerator/Database/Migrations/2018_05_25_203256_create_report_formats_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * This file creates draggable_components for the report-generator.
```

Comment:
It should be "This file created report_format for report-generator." . Just a guess.

Reply:
Checked!